torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1994,6 +1999,10 @@ def _make_tensor_helper(shape, low=None, high=None):
            dtypes=all_types_and(torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_floor_divide,
            decorators=[_wrap_warn_once("floor_divide is deprecated, and will be removed")],
+           skips=(
+               # `test_duplicate_method_tests` doesn't raise any warning, as it doesn't actually
+               # call the operator.
+               SkipInfo('TestOpInfo', 'test_duplicate_method_tests'),),
```

Comment:
@mruberry, think there is a nicer way to avoid the skip?

Reply:
No, this seems fine, although it is a humorous niche interaction.